demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -276,19 +304,36 @@ def _collect_playbooks_dependencies(pack_playbooks, id_set):
                                                                                              id_set['integrations']) \
                     if integration_name else PackDependencies._search_packs_by_integration_command(command, id_set)
 
+                # ---- integrations packs ----
                 if packs_found_from_integration:
                     pack_dependencies_data = PackDependencies._detect_generic_commands_dependencies(
                         packs_found_from_integration)
                     dependencies_packs.update(pack_dependencies_data)
 
+            # ---- other playbooks packs ----
             implementing_playbook_names = playbook_data.get('implementing_playbooks', [])
             packs_found_from_playbooks = PackDependencies._search_packs_by_items_names(implementing_playbook_names,
                                                                                        id_set['playbooks'])
-
             if packs_found_from_playbooks:
                 pack_dependencies_data = PackDependencies._label_as_mandatory(packs_found_from_playbooks)
                 dependencies_packs.update(pack_dependencies_data)
 
+            # ---- incident fields packs ----
+            incident_fields = playbook_data.get('incident_fields', [])
+            packs_found_from_incident_fields = PackDependencies._search_packs_by_fields_names(incident_fields,
```

Comment:
Why all incident fields have `incident_` prefix in id-set and in `incident_fields` list in playbooks it appears without it?

Reply:
Thats exactly the reason I had to write another method.
In the playbook, the field will be `email` while in the id set it will be under `incident_email`
I could manipulate the name in the id_set creation but I figured I can do it here